crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -66,10 +66,16 @@ pub enum State {
     Error,
 }
 
-/// A supervisor that starts two sync tasks, one for the room list and one for
-/// the end-to-end encryption support.
+/// A supervisor responsible for managing two sync tasks: one for handling the
+/// room list and another for supporting end-to-end encryption.
+///
+/// The two sync tasks are spawned as child tasks and are contained within t
```

Comment:
```diff
- within t
+ within the
```

Reply:
Thanks for spotting this: https://github.com/matrix-org/matrix-rust-sdk/pull/4543/commits/f4c94abe471d89432bf61df12a9fe92ea2309682.